libs/jwst-rpc/src/client.rs

```diff
@@ -135,13 +135,17 @@ async fn run_sync(
 fn start_sync_thread(workspace: &Workspace, remote: String, mut rx: Receiver<Vec<u8>>) {
     debug!("spawn sync thread");
     let first_sync = Arc::new(AtomicBool::new(false));
-    let first_sync_cloned = first_sync.clone();
+    let [first_sync_cloned, first_sync_cloned2] = [first_sync.clone(), first_sync.clone()];
```

Comment:
we can put the second clone in spawn to make the code easier to read

Reply:
updated